TERMS_AND_CONDITIONS.md

```diff
@@ -0,0 +1,73 @@
+## TERMS & CONDITIONS
+_Last updated: August 26, 2019_
+
+Please read these Terms and Conditions carefully before using the Skybunk mobile application operated by Project Stairwell.
+
+Your access to and use of the Service is conditioned on your acceptance of and compliance with this Agreement. The Agreement applies to all visitors, users and others who access or use the Application.
+
+*BY DOWNLOADING AND USING THE APPLICATION, YOU ARE AGREEING TO THE FOLLOWING:*
+
+* I understand that I should not post any personal information other than my full name, and that I should use a completely original password.
+* I understand that using a fake name can result in account deletion and restricted or terminated access to the Application
+* I agree to all of the Terms and Conditions outlined in the Agreement.
+
+**1. Definitions**
+In this Agreement, unless the context clearly indicates otherwise, the following words and expressions shall have the meaning ascribed:
+
+1.1 “Agreement” means this document containing terms and conditions.
+
+1.2 “Application” refers to the Skybunk website and mobile app on both Android and iOS devices.
+
+1.3 “Content Administrator” refers to the person or person(s) appointed by your residence to moderate content and remove any content that violates these terms or other terms you have agreed to in your residence contract.
+
+1.4 “Institution” refers to the host community of a specific instance of the Application.
+
+1.5 “Project Stairwell” refers to a group of students and alumni who maintain the Application.
+
+1.6 “User(s)” refer to anyone who holds an account with the Application.
+
+**2. Profile**
+
+In order to maintain a respectful community, the User profile must never contain:
+
+* A name other than the one used in everyday life.
+* Inaccurate information about the user.
+* An offensive or otherwise inappropriate photo as the profile picture.
+
+
+**3. Content Regulations**
+
+Skybunk is a vessel in which Users associated with an Institution can communicate and interact. You may NOT post content:
+
+That is unlawful, misleading, discriminatory or fraudulent.
```

Comment:
Small formatting note, but this should probably be a bulletted list

Reply:
Thanks for the catch! My formatting didn't properly convert to markdown so there were a bunch of things that didn't transfer :(